packages/datagateway-common/src/state/reducers/app.reducer.tsx

```diff
@@ -0,0 +1,14 @@
+import { combineReducers, Reducer } from 'redux';
```

Comment:
Pretty sure this file isn't needed or used since `datagateway-common` never actually sets up it's own store - it's expected that the plugins will do this instead.